kayenta-core/kayenta-core.gradle

```diff
@@ -1,40 +1,27 @@
 dependencies {
-  compile spinnaker.dependency('jedis')
-  compile spinnaker.dependency('kork')
-  compile spinnaker.dependency('korkJedisTest')
-  compile spinnaker.dependency("korkSwagger")
-  compile spinnaker.dependency('lombok')
-  compile spinnaker.dependency('spectatorApi')
+  implementation "redis.clients:jedis"
+  implementation "com.netflix.spinnaker.kork:kork-core"
+  implementation "com.netflix.spinnaker.kork:kork-web"
+  implementation "com.netflix.spectator:spectator-api"
 
   // TODO update korkSwagger?
   // Force a newer version than what comes with korkSwagger, so that the api docs page isn't broken.
```

Comment:
Is this still true with the Kork dep changes?

https://github.com/spinnaker/kork/commit/6d1cb51d652dcc1be1435bfb9ba8d28b3addefcb#diff-2ac723eaa3292cedf317c0d9e863ac20

Looks like springfox was ripped out and replaced, I wonder if that has to do with the swagger page not loading?